grakn-engine/src/main/java/ai/grakn/engine/controller/TasksController.java

```diff
@@ -165,54 +188,190 @@ private Json createTask(Request request, Response response) {
         String intervalParam = request.queryParams(REST.Request.TASK_RUN_INTERVAL_PARAMETER);
         String priorityParam = request.queryParams(REST.Request.TASK_PRIORITY_PARAMETER);
 
+        TaskState taskState = processTask(className, createdBy, runAtTime, intervalParam,
+                priorityParam);
+        manager.addTask(taskState, TaskConfiguration.of(bodyAsJson(request)));
+        // Configure the response
+        response.type(ContentType.APPLICATION_JSON.getMimeType());
+        response.status(200);
+
+        return Json.object("id", taskState.getId().getValue());
+    }
+
+    @POST
+    @Path("/bulk")
+    @ApiOperation(value = "Schedule a set of tasks.")
+    @ApiImplicitParams({
+            @ApiImplicitParam(name = REST.Request.TASKS_PARAM, value = "JSON Array containing an ordered list of tasks (see single task endpoint). The response references the order the task appears in the list as an index.", required = true, dataType = "List", paramType = "body"),
```

Comment:
That's right, I think it's because when I started working on this I thought the relevant info to instantiate the task was in the task parameters. What's the distinction between config and parameters? If we are refactoring the whole endpoint it might make sense to rethink that.